### ISSUE TITLE
gdal doesn't build python bindings

### ISSUE THREAD

**Issue by JensTimmerman** [hpcugent/easybuild-easyconfigs#3396]
currently, python is not a dependency of GDAL, and as such, GDAL does not build it's python bindings, and doesn't isntall tools like `gdal_merge.py`

**Comment by boegel** [comment]
it's not so much that there's no Python dependency specified, but that the `--with-python` configure option is not passed

However, trying that, I'm running into this weird problem:

```
/bin/sh: -d: invalid option                                                                                                                                                                                                                                                                                            error: command '/bin/sh' failed with exit status 1 
```

Another option is to install the GDAL Python bindings stand-alone via https://pypi.python.org/pypi/GDAL

cc @wpoely86

**Comment by boegel** [comment]
This page provides some useful information: http://www.michael-joost.de/gdal_install.html

Apparently defining `$CC` & co makes the install procedure go haywire... :-/

**Comment by boegel** [comment]
The problem can be circumvented relatively easily by also using `--without-libtool` (hat tip @wpoely86), but then the `libgdal.a` library is not being installed (the sanity check fails).

@wpoely86 is that the problem you ran into as well?

**Comment by wpoely86** [comment]
No, I really hit a hard wall when I tried. Cannot remember what exactly.

**Comment by boegel** [comment]
fixed with #3413, we already had a GDAL easyconfig that included the Python tools, where `--without-libtool` was being used to dance around the `invalid option` issue...
